Gemfile

```diff
@@ -1,5 +1,4 @@
 source 'https://rubygems.org'
-ruby '2.0.0'
```

Comment:
As Travis CI is set to test the project using different ruby versions it complains when the ruby version differ from the one set in the Gemfile.

"Your Ruby version is 1.8.7, but your Gemfile specified 2.0.0"

Not sure if it's the best solution